### PR TITLE
Adding support for Angular 11

### PR DESCRIPTION
**Pull request by pheinicke**
- also updating dependencies, fixing vulnerabilities

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -60,59 +60,60 @@
...
     },
     "dependencies": {
-        "tslib": "^2.0.0"
+        "tslib": "^2.0.3"
     },
     "peerDependencies": {
-        "@angular/common": ">=10.0.0 <11.0.0",
-        "@angular/core": ">=10.0.0 <11.0.0",
-        "@angular/platform-browser": ">=10.0.0 <11.0.0",
+        "@angular/common": ">=11.0.0 <12.0.0",
```
Comment (Romanchuk) [inline package.json:66]: We shouldn't raise min version, since libary has no braking changes to support angular 10.
>=10.0.0 <12.0.0

**Review by Romanchuk — CHANGES_REQUESTED** [review]
@pheinicke Thx for your contribution!

File: package.json (hunk 1 above)
Comment (pheinicke) [reply package.json]: Done

**Review by Romanchuk — APPROVED** [review]

**Comment by Romanchuk** [comment]
@pheinicke I'm going to release beta on monday-tuesday, check it on demo app, and if it ok, release stable version

**Pull request merged**